core/domain/topic_domain.py

```diff
@@ -114,6 +119,13 @@ def __init__(self, change_dict):
             self.property_name = change_dict['property_name']
             self.new_value = copy.deepcopy(change_dict['new_value'])
             self.old_value = copy.deepcopy(change_dict['old_value'])
+        elif self.cmd == CMD_UPDATE_SUBTOPIC_PAGE_PROPERTY:
```

Comment:
This seems like it's in the wrong place. Subtopic page content is the responsibility of the SubtopicPageModel and should have its own type of change. Treat topic and subtopic-page models as independent, with one linking to the other (similar to collections and the explorations they contain).

Reply:
Done, moved it into subtopic_page_services